src/Pages/Main/Components/Nav/Nav.scss

```
@@ -129,45 +129,89 @@ a {
     position: relative;
   }
 
-  .searchContainer {
+  .formGroup {
+    padding-top: 5px;
     input {
       margin: 0px 10px;
       outline: 0px;
       border: none;
       border-bottom: $lightGrey solid 1px;
     }
+  }
 
-    &:hover {
-      .searchTrend {
-        display: block;
-      }
+  .searchButtonIcon {
+    i {
+      padding-left: 10px;
+    }
+  }
+
+  .searchAndRecentContainer {
+    position: absolute;
+    margin-top: 37px;
+    right: 0px;
+    display: flex;
+    border-radius: 15px;
+  }
+
+  .searchContainer:hover {
+    position: relative;
+    padding: 40px 0px 40px 0px;
```

```suggestion
    padding: 40px 0px;
```
단축 속성 사용할 수 있겠네요! 유사한 부분들이 보이는데 역시 수정해주세요.

---

넵! 다른 곳에서도 단축 속성 사용해주었습니다!